src/components/Header/Header.tsx

```diff
@@ -108,34 +111,47 @@ const Header = () => {
   };
 
   if (authState.requestToken) {
-    window.location.assign('http://www.google.pt');
+    const redirect = 'http://localhost:3000/approved';
```

Comment:
We need to find another way to get the URL, as this can't be hardcode. Maybe use the .env file to set this variable

Reply:
Done 👍 Created AUTH_REDIRECT_URL variable for that.